.github/ISSUE_TEMPLATE/config.yml

```diff
@@ -0,0 +1,11 @@
+blank_issues_enabled: false
+contact_links:
+  - name: Feature Request
+    url: https://forem.dev/t/features
+    about: Please use the `Create Post` feature request template to suggest an idea to make Forem better. 
+  - name: Forem Questions and Discussions
+    url: https://forem.dev/
+    about: Please ask and answer questions, discuss features, or reach out for help on forem.dev
+  - name: Self Host Bug Report
+    about: Create a report to help us improve the Self Host repository.
```

Comment:
**(nitpick, non-blocking)**: Can we swap the order of the `about` and `url` keys  in the `Self Host Bug Report` to keep it consistent with the order of the other points?

Reply:
Oops! I totally missed that. Yeah I think it should be swapped anyway. Thanks for catching that. I switched it.